includes/text-image-list.html

```diff
@@ -0,0 +1,35 @@
+{%- assign textImageList = include.textImages -%}
+{%- assign textImageListHeadline = include.headline -%}
+{%- assign textImageListHeadlineLevel = include.level | default: 'h2' -%}
+{%- assign textImageListHeadlineColor = include.headlineColor -%}
+{%- assign textImageListBgColor = include.bgColor | default: false -%}
+
+<text-image-list
+  headline="{{ textImageListHeadline }}"
+  spacing="{{ textImageListHeadlineSpacing }}"
+  bg-color="{{ textImageListBgColor }}"
+  headline-color="{{ textImageListHeadlineColor }}"
+>
+
+  {% assign textImageListSize = textImageList | size %}
+  {% for textImage in textImageList %}
```

Comment:
Versuch deine Variablen-Namen hier entsprechend der Komponente zu prefixen. Ansonsten kommst du leider in Konflikt mit der Komponente "text-image"